roop/processors/frame/face_enhancer.py

```diff
@@ -43,58 +45,30 @@ def get_code_former():
             CODE_FORMER.load_state_dict(model)
             CODE_FORMER.eval()
         return CODE_FORMER
-    
-
-def get_face_enhancer():
-    global FACE_ENHANCER
-
-    with THREAD_LOCK:
-        if FACE_ENHANCER is None:
-            FACE_ENHANCER = FaceRestoreHelper(
-                upscale_factor=int(2),
-                face_size=512,
-                crop_ratio=(1, 1),
-                det_model='retinaface_resnet50',
-                save_ext='png',
-                use_parse=True,
-                device='cuda'
-            )
-    return FACE_ENHANCER
-
-
-def enhance_face_in_frame(cropped_faces):
-    faces_enhanced = []
-    for _, cropped_face in enumerate(cropped_faces):
-        face_in_tensor = normalize_face(cropped_face)
-        face_enhanced = restore_face(face_in_tensor)
-        faces_enhanced.append(face_enhanced)
-    return faces_enhanced
 
 
-def process_frame(source_face: any, temp_frame: any) -> any:
-    THREAD_SEMAPHORE.acquire()
-    face_helper = get_face_enhancer()
-    face_helper.read_image(temp_frame)
-    # get face landmarks for each face
-    face_helper.get_face_landmarks_5(only_center_face=False, resize=640, eye_dist_threshold=5)
-    # align and warp each face
-    face_helper.align_warp_face()
-    cropped_faces = face_helper.cropped_faces
-    faces_enhanced = enhance_face_in_frame(cropped_faces)
-    for face_enhanced in faces_enhanced:
-        face_helper.add_restored_face(face_enhanced)
-    face_helper.get_inverse_affine()
-    result = face_helper.paste_faces_to_input_image()
-    face_helper.clean_all()
-    THREAD_SEMAPHORE.release()
-    return result
-
+def align_warp_face(temp_frame):
+    many_faces = get_many_faces(temp_frame)
+    if many_faces:
+        face_list = []
+        matrix_list = []
+        for target_face in many_faces:
+            face_long_side = max(target_face.bbox[2] - target_face.bbox[0], target_face.bbox[3] - target_face.bbox[1])
+            face_size = int(np.ceil(face_long_side / 128) * 128)
```

Comment:
there are many magic numbers here to be honest - their meaning is not self explaining